### ISSUE TITLE
PySide2 application fail to run if the exe file path include non-ascii characters

### ISSUE THREAD

**Issue by count10GH** [marcelotduarte/cx_Freeze#593]
Windows 10 Japanese/ Python 3.7 / PySide2 5.14 / cx_Freeze 6.1 

Built PySide2 GUI application fail to run with showing dialog with message
"This application failed to start because no QT platform plugin could be initialized, Reinstalling the application may fix this problem."
when exe file path is "D:\Temp\日本語\build\exe.win-amd64-3.7\GreetingGUI.exe".
When path is "D:\Temp\working\build\exe.win-amd64-3.7\GreetingGUI.exe", run properly.

test code
```
#!/bin/env python
# coding: utf-8
import sys
import PySide2.QtWidgets as QtWidgets

app = QtWidgets.QApplication(sys.argv)

mainwindow = QtWidgets.QWidget()
mainwindow.setLayout(QtWidgets.QVBoxLayout())

label = QtWidgets.QLabel(mainwindow)
label.setText('Hello World!!')
label.parent().layout().addWidget(label)

mainwindow.show()

sys.exit(app.exec_())
```
setup.py
```
from cx_Freeze import setup, Executable
import platform

# Dependencies are automatically detected, but it might need
# fine tuning.
buildOptions = dict(
    packages = [],
    excludes = [],
    includes = [],
)

base = 'Console'
if platform.system().lower() == 'windows':
  base = 'Win32GUI'

executables = [
    Executable('sayHelloGUI.py', base=base, targetName = 'GreetingsGUI')
]

setup(name='SayHello',
      version = '1.0',
      description = 'print greetings',
      options = dict(build_exe = buildOptions),
      executables = executables)
```

**Comment by marcelotduarte** [comment]
I tested your sample on linux. 
Without freezing:
python sayHelloGUI.py
on ascii folder: ok
on japanse folder: 
```
testes/日本語$ python sayHelloGUI.py
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized.
Reinstalling the application may fix this problem.
```
My pip list:
```
Package       Version
------------- -------
cx-Freeze     6.1    
pip           20.0.2 
pkg-resources 0.0.0  
PySide2       5.14.1 
setuptools    45.2.0 
shiboken2     5.14.1 
```
Freezing, it runs on ascii folder, and core dump at japanse folder (I see a problem with sys.path surrogates, but this is another issue).
So, I think that the issue is with pyside2 / shiboken2.
Please test it on normal python or adjust the sample on normal python first, and the we can continue to explore the issues at frozen side.

**Comment by marcelotduarte** [comment]
@count10GH With the new version of PySide2 5.14.2.1 I tested it:
- pure python (not frozen)
the folder in ASCII - ok
the folder in Portuguese, virtual env name in Portuguese - ok 
the folder in Japanese, virtual env name in ASCII - ok
the folder in Japanese, virtual env name in Japanese - fail
- now, frozen:
source\build\exe..... where source folder in ASCII - ok
source\build\exe..... where source folder in Portuguese - ok (in previous version it fails)
source\build\exe..... where source folder in Japanese - fail

Between versions 5.14.1 and 5.14.2.1 have some progress and I can conclude that the issue is at the pyside2 side. Before, In 5.14.1 it runs only with the folder in pure ASCII, in python or frozen.

See related issue #638 for using py38.

**Comment by marcelotduarte** [comment]
cx_Freeze 6.2 has just been released. Please, try it out.

**Comment by marcelotduarte** [comment]
cx_Freeze 6.6 has just been released.
Can you check if this has been resolved?

**Comment by marcelotduarte** [comment]
cx_Freeze 6.7b2 has just been released with a fix for windows that is relevant (#1077).
pip install cx-Freeze==6.7b2

Closing due to lack of response.
If you had issues please open a new issue.
